core/publish.gradle

```diff
@@ -26,16 +26,18 @@ afterEvaluate {
         }
         publications {
             release(MavenPublication) {
+                from components.release
+
                 groupId = releaseConfig.groupId
                 version = versions.goldfinger
 
                 artifactId = 'goldfinger'
 
-                artifact bundleReleaseAar
```

Comment:
Is there a need to remove this and add
`packaging = "aar"` in pom block?
Both modules are android libraries (not java libraries) 

If nobody knows I can recheck it

Reply:
Could you please recheck what is the core difference between those two? 
I found that `packaging = "aar"` is used as metadata to tell the consumers that this is aar, but `artifact bundleReleaseAar` specifies the file to publish. I think that we might want to use a combination of both.

Reply:
Yea, sure I will look into it when I get time

Reply:
When using `artifact bundleReleaseAar` I was actually getting error.
But removing it, and removing `packaging = "aar"` resulted in same pom file.
So option `packaging = "aar"` doesn't seem to do anything 🤷‍♂️ 
But we can leave it, no need to change anything now